### DISCUSSION TITLE
Updating spec of already completed feature?

### DISCUSSION THREAD

**Discussion by Trenly** [microsoft/winget-cli#1285] [Q&A] [answered]
I was looking at the spec for the [Settings command](https://github.com/microsoft/winget-cli/blob/master/doc/specs/%23396%20-%20Settings%20command.md) and noticed a few places where it could be clarified or brought up to date with other documentation. Would it be appropriate to retro-actively update the specification to align with the final implementation and resolve clarity issues?

**Comment by denelon** [comment, marked as answer]
@Trenly that's a great question. We use the specifications to guide implementation. Clearly we have some drift. Do you see those discrepancies on Microsoft Docs, or just the specification?

**Reply by Trenly** [reply]
Just in the specification, as far as I can tell

**Reply by denelon** [reply]
@Trenly, go ahead and make a draft PR, and mention me so I can review. Since it's only helping with understanding, I will likely approve.

**Reply by Trenly** [reply]
I've created #1290 as a draft

**Reply by denelon** [reply]
Awesome! Thanks for the help with documentation.
